docs/contracts_04_tx_execution.md

```diff
@@ -10,7 +10,7 @@ To execute a transaction with the Gnosis Safe the <span style="color:#DB3A3D">`e
 - _safeTxGas_: This is the minimum amount of gas that is provided for the Safe transaction. In case of <span style="color:#DB3A3D">`CALL`</span> and <span style="color:#DB3A3D">`DELEGATECALL`</span> this is also the maximum available gas (gas limit).
 - _baseGas_: This is the amount of gas that is independent of the specific Safe transactions, but used for general things such as signature checks and the base transaction fee. _SafeTxGas_ and _baseGas_ combined are comparable to the gas limit of a regular transaction.
 - _gasPrice_: Same like for a regular Ethereum transaction. Setting the gas price to 0 means that no refund is paid out.
-- _gasToken_: For regular Ethereum transactions, gas is paid in ether, always. The Gnosis Safe enables users to pay in ERC20 tokens or ether. The desired token is specified here. If <span style="color:#DB3A3D">`0x0`</span> then Ether is used. Gas costs are calculated by <span style="color:#DB3A3D">`(dataGas + txGas) * gasPrice`
+- _gasToken_: For regular Ethereum transactions, gas is paid in ether, always. The Gnosis Safe enables users to pay in ERC20 tokens or ether. The desired token is specified here. If <span style="color:#DB3A3D">`0x0`</span> then Ether is used. Gas costs are calculated by <span style="color:#DB3A3D">`(baseGas + txGas) * gasPrice`
```

Comment:
we are missing a closing `span` here, could you fish it too?

Reply:
fixed it